apps/condo/domains/news/access/GetNewsSharingRecipientsCountersService.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Generated by `createservice news.GetNewsItemSharingsRecipientsCountersService --type queries`
+ */
+const get = require('lodash/get')
+
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+const { getById } = require('@open-condo/keystone/schema')
+
+const { checkPermissionsInEmployedOrRelatedOrganizations } = require('@condo/domains/organization/utils/accessSchema')
+
+async function canGetNewsSharingRecipientsCounters ({ authentication: { item: user }, context, args: { data: { b2bAppContext: { id: b2bAppContextId } } } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+    if (user.isAdmin || user.isSupport) return true
+
+    const b2bContext = await getById('B2BAppContext', b2bAppContextId)
+    if (!b2bContext) return false
+
+    const organizationId = get(b2bContext, 'organization', null)
+    if (!organizationId) return false
+
+    return await checkPermissionsInEmployedOrRelatedOrganizations(context, user, organizationId, 'canManageNewsItems')
```

Comment:
Maybe `canReadNewsItems`?
If I only have read access to news then i should see this counter too

Reply:
Ok, I get it
This block should not be displayed there without write permissions